include/open62541/server.h

```diff
@@ -711,6 +711,13 @@ UA_Server_readAccessLevel(UA_Server *server, const UA_NodeId nodeId,
                             outAccessLevel);
 }
 
+static UA_INLINE UA_THREADSAFE UA_StatusCode
+UA_Server_readUserAccessLevel(UA_Server *server, const UA_NodeId nodeId,
```

Comment:
This method does not make any sense.
Because the UserAccessLevel depends on the session.
But if you locally read the UserAccessLevel then you do it outside the context of a session.
You would get back the exact same result as for `UA_Server_readAccessLevel`.

I would prefer if you left a comment why there is no `UA_Server_readUserAccessLevel`.